blockchain/tx_pool.go

```diff
@@ -655,24 +658,38 @@ func (pool *TxPool) validateTx(tx *types.Transaction) error {
 			return ErrFeeCapVeryHigh
 		}
 
+		// TODO : Need to KIP-71 hardfork.
 		// Ensure gasFeeCap is greater than or equal to gasTipCap.
-		if tx.GasFeeCap().Cmp(tx.GasTipCap()) < 0 {
-			return ErrTipAboveFeeCap
-		}
-
-		if pool.gasPrice.Cmp(tx.GasTipCap()) != 0 {
-			logger.Trace("fail to validate maxPriorityFeePerGas", "unitprice", pool.gasPrice, "maxPriorityFeePerGas", tx.GasFeeCap())
-			return ErrInvalidGasTipCap
-		}
-
-		if pool.gasPrice.Cmp(tx.GasFeeCap()) != 0 {
-			logger.Trace("fail to validate maxFeePerGas", "unitprice", pool.gasPrice, "maxFeePerGas", tx.GasTipCap())
-			return ErrInvalidGasFeeCap
+		//if tx.GasFeeCap().Cmp(tx.GasTipCap()) < 0 {
+		//	return ErrTipAboveFeeCap
+		//}
+		//
```

Comment:
Do we need to remove this condition?

Reply:
I also removed this condition because `GasTipCap` field will be ignored.
Would you explain to me why this condition is needed?